README.md

```diff
@@ -35,3 +35,95 @@ console.log(binaryPacket.toString('hex'))
 
 const jsonPacket = packet.deserializeIlpPayment(binaryPacket)
 ```
+
+### Serialize/deserialize ILQP Quote Requests/Responses
+
+#### QuoteLiquidityRequest
+
+```js
+const packet = require('ilp-packet')
+
+const binary = packet.serializeIlqpLiquidityRequest({
+  requestId: 'c217449a-69e9-4a08-9575-6e7eafa818c0',
+  destinationAccount: 'example.nexus.bob',
```

Comment:
> The response always tells you what you wanted to know. What are you talking about?

"destinationPrefix" made me think the quote was "only" up to that prefix, meaning the connector was giving you _less_ information than you asked for rather than _more_. I think I understand it better now.

> This is going to be an extremely common case (basically all recurring payments) because we do things like append unique IDs to ILP addresses.

We're no longer doing this, now that the UUIDs for payments can go in the transport headers. We still have the receiver ID in the address, but there isn't anything that's payment-specific there.

> The use case is: example.ledger1.alice wants to send 100 payments to individual accounts example.blah.dest.bob1, example.blah.dest.bob2, example.blah.dest.bob3 etc. So to start, she asks for a quote for destination account example.blah.dest.bob1

Why doesn't Alice ask for a quote to `example.blah.dest.` if she knows all of those accounts share a prefix? Or is the idea that she doesn't _really_ know if all of them will have the same rates until the connector tells her that?

Given this explanation, I'm understanding this a bit differently. Am I right in thinking that if I ask for a quote to a specific account I could ignore the prefix altogether and just use the curve? It sounds like the prefix is just extra info that I may want to use to avoid making subsequent quote requests. If that's the case, what about a name like `appliesToPrefix`? If you don't want to rename it, this needs a very good explanation to go along with it that makes it clear what this field means and how you're supposed to use it.